package.json

```diff
@@ -42,7 +42,7 @@
     "lodash": "4.17.15",
     "minimist": "^1.2.0",
     "murmur-hash-js": "1.0.0",
-    "node-elm-compiler": "5.0.3",
+    "node-elm-compiler": "5.0.4",
```

Comment:
Doesn't it make sense to make it `~5.0.4` so we get patch updates without a new release?

Reply:
I think the `package-lock.json` file will pin it down anyway. I think that's intentional, for applications to pin dependencies so they don't accidentally break when some dependency does a patch update that turns out not be a patch update. Don't have a super strong opinion though and just kept this the way it was. Happy to change it!

Reply:
If I understand https://docs.npmjs.com/files/package-lock.json it seems like it will be ignored: 

> One key detail about package-lock.json is that it cannot be published, and it will be ignored if found in any place other than the toplevel package.

So if you keep a package-lock.json in your project, it will ensure things don't accidentally get updated unless you want it to.

Reply:
I did not know that, thanks for sharing!

I'm not sure what to do here. Almost all packages in the package.json are currently pinned, so I feel maybe this is a broader question? I'm currently inclined to leave it pinned on this PR, and maybe we can create a separate PR/issue to propose using `~` for all dependencies?